apps/costcontrol/style/views/datausage.css

```diff
@@ -92,27 +104,24 @@
 }
 
 #datausage-tab .app-item {
-  width: 32rem;
```

Comment:
I don't feel confident about that change, at the first look I can't imagine the case when we need this fixed width, but I'm not familiar with CostControl app enough, so I'll let you decide if this what we need :)